runtime/shared_common/shrinit.cpp

```diff
@@ -3851,7 +3881,10 @@ j9shr_print_stats(J9JavaVM *vm, UDATA parseResult, U_64 runtimeFlags, UDATA prin
 			RESULT_DO_PRINTALLSTATS_EQUALS == parseResult||
```

Comment:
might as well fix the formatting for all of the conditions

Reply:
Fixed